packages/presentation/src/file.ts

```diff
@@ -284,3 +284,18 @@ async function uploadFileWithSignedUrl (file: File, uuid: string, uploadUrl: str
     })
   }
 }
+
+export async function fetchJson (file: string, name: string): Promise<Blob | undefined> {
```

Comment:
`json()` returns `any`, not `Blob`
https://developer.mozilla.org/en-US/docs/Web/API/Response/json

Reply:
Thanks, fixed.